solutions/models.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+from django_nanoid.models import NANOIDField 
+from utils.choices import Sender
+
+# Create your models here.
+
+class SolutionHistory(models.Model):
+    id = NANOIDField(
+    primary_key=True,
+    editable=False,
+    unique=True,
+    max_length=21,
+    )
+    user=models.ForeignKey('accounts.User', on_delete=models.CASCADE, related_name='solutionhistory')
+    user=models.ForeignKey('accounts.User', on_delete=models.CASCADE, related_name='solution_history')
+    mission=models.ForeignKey('missions.Mission', on_delete=models.CASCADE, related_name='solution_history')
```

Comment:
❗ 제가 [이전 코드 리뷰](https://github.com/VibeNext/Back/pull/5#discussion_r2497843230)에서 라인을 잘못 선택했었네요, 수정 부탁드립니다.
```suggestion
    user=models.ForeignKey('accounts.User', on_delete=models.CASCADE, related_name='solution_history')
    mission=models.ForeignKey('missions.Mission', on_delete=models.CASCADE, related_name='solution_history')
```